config/default.js

```diff
@@ -288,4 +289,9 @@ module.exports = {
 
   // If true, show a link to the beta versions page on the listing page.
   betaVersions: true,
+
+  // Show and link to User Profiles using the front-end. These pages
+  // are still in development so this should only be enabled locally and
+  // on `-dev`.
+  enableUserProfile: false,
```

Comment:
Using config flags like this is a great way to ship half-working things. It helps a lot because we can iterate in smaller patches which are easier to review. Your patch here is already small, I just wanted to encourage this type of approach for future patches.